srcpkgs/OpenBoardView/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'OpenBoardView'
+pkgname=OpenBoardView
+version=9.95.1
+revision=1
+create_wrksrc=yes
+build_style=cmake
+hostmakedepends="pkg-config python3"
+makedepends="SDL2-devel gtk+3-devel fontconfig-devel
+ zlib-devel"
+depends="sqlite $(vopt_if mdbtools)"
```

Comment:
`vopt_if` is missing an argument.

Reply:
I removed mdbtools from depends, OpenBoardView doesn't use it, it's used by separate shell script utility.

https://github.com/OpenBoardView/OpenBoardView/blob/master/utilities/bvconv.sh